### ISSUE TITLE
Make meta data enrichment of uploaded files configurable

### ISSUE THREAD

**Issue by hmccloy** [marketing-factory/ai-filemetadata#15]
In some cases it's not desired that alt text is added automatically to files. For this cases it should be possible to disable the enrichment of uploaded files.

**Comment by ischmittis** [comment]
See #3

**Comment by Idleworks** [comment]
See PR #40

**Comment by ischmittis** [comment]
Closed by https://github.com/marketing-factory/ai-filemetadata/pull/40
